tests/unit/spec/firefox/landing/partner-build-download.js

```diff
@@ -0,0 +1,265 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+/* For reference read the Jasmine and Sinon docs
+ * Jasmine docs: https://jasmine.github.io/
+ * Sinon docs: http://sinonjs.org/docs/
+ */
+
+import PartnerBuildDownload from '../../../../../media/js/firefox/landing/partner-build-download.es6';
+
+describe('partner-build-download.es6.js', function () {
+    describe('init()', function () {
+        beforeEach(function () {
+            const strings = `<div id="strings" data-win-custom-id="partner-firefox-release-smi-smi-001-stub" data-mac-custom-id="partner-firefox-release-smi-smi-001-latest"></div>`;
+
+            const button1 = `<div id="download-primary" class="download-button mzp-c-button-download-container">
+                <ul class="download-list">
+                    <li class="os_win64">
+                        <a class="download-link button" id="download-primary-win64" href="https://download.mozilla.org/?product=firefox-stub&amp;os=win64&amp;lang=en-GB" data-download-version="win64">
+                            Download Firefox
+                        </a>
+                    </li>
+                    <li class="os_win64-msi">
+                        <a class="download-link button" id="download-primary-win64-msi" href="https://download.mozilla.org/?product=firefox-msi-latest-ssl&amp;os=win64&amp;lang=en-GB" data-download-version="win64-msi">
+                            >Download Firefox
```

Comment:
Was that mean to be?

```suggestion
                            Download Firefox
```